src/Toml/Bi/Combinators.hs

```diff
@@ -170,7 +147,7 @@ mdimap toString toMaybe bi = Bijection
   , biWrite = \s -> do
         retS <- biWrite bi $ toString s
         case toMaybe retS of
-            Nothing -> throwError $ DecodeParsing "Unable to convert" -- TODO
+            Nothing -> error $ "Given pair of functions for 'mdimap' doesn't satisfy roundtrip property"
```

Comment:
Probably highlight this behavior in the haddock and say what property should be satisfied for these functions?